api/Element.json

```
@@ -5957,6 +5957,54 @@
           }
         }
       },
+      "nextElementSibling": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Element/nextElementSibling",
+          "support": {
+            "chrome": {
+              "version_added": "4"
```

```suggestion
              "version_added": "1"
```

Based on collector results using http://mdn-bcd-collector.appspot.com/tests/api/Element/nextElementSibling.